src/Rule/EnumMulti.php

```diff
@@ -53,7 +53,7 @@ public function setParameter($params)
 	public function getMessageParameters()
 	{
 		return array(
-			'values' => implode('|', $this->getParameter()),
+			'value' => implode('|', $this->getParameter()),
```

Comment:
I would like to change from `value` to `params` etc, but then `value` is still an array but how do you solution this?